docs/concepts/operator.md

```diff
@@ -153,10 +154,29 @@ Each one is described in more details below.
 - `.spec.deployment.extension.helm` and `.spec.deployment.extension.values` are used when creating the `ControllerDeployment` in the garden cluster.
 - `.spec.deployment.extension.policy` and `.spec.deployment.extension.seedSelector` define the extension's installation policy as per the [`ControllerDeployment's` respective fields](../extensions/controllerregistration.md#deployment-configuration-options)
 
-The extension controller can also be deployed in the runtime cluster to manage resources required by the `Garden` resource.
+##### Runtime
+
+The `extension` controller can be deployed in the runtime cluster to manage resources required by the `Garden` resource (e.g. `BackupBucket`, `DNSRecord`, `Extension`).
 Since the environment in the runtime cluster may differ from that of a `Seed`, the extension is installed in the runtime cluster with a distinct set of Helm chart values specified in `.spec.deployment.extension.runtimeValues`.
+If no `runtimeValues` are not given, the extension deployment for the runtime garden is considered superfluous and the deployment is skipped or uninstalled.
 This configuration allows for precise control over various extension parameters, such as requested resources, [priority classes](../development/priority-classes.md), and more.
 
+Besides the values configure in `.spec.deployment.extension.runtimeValues`, a runtime deployment indicator is merged into the values:
+
+```yaml
+gardener:
+  runtimeCluster:
+    enabled: true # indicates the extension deployment is responsible for the Garden cluster, e.g. for handling `BackupBucket`, `DNSRecord` and `Extension` objects.
```

Comment:
Please consider rewording the sentence. How is the "extension deployment" responsible for anything?